.github/workflows/cypress.yaml

```diff
@@ -289,5 +292,5 @@ jobs:
                   product: "Element Web"
                   product-version: ${{ github.event.workflow_run.head_branch }}
                   build-id: ${{ github.event.workflow_run.head_sha }}
-                  suite-name: "Cypress E2E"
+                  suite-name: "Cypress E2E (${{ matrix.crypto }} crypto)"
```

Comment:
So actually i think we want this chnge on the build-id rather than the suite name; that would get us the same suite running against two versions (with/without rust) rather than two separate suites. Something like

Both suites will have the same tests in them so that shouldn't be a problem to have two builds going on here.

Reply:
@michaelkaye thanks. something like this? (eeffc8a)